### PR TITLE
[stable/drupal] Simplify ingress configuration for cert-manager

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juan@bitnami.com>

### What this PR does / why we need it:

This PR allows the user to configure the proper annotations when using cert-manager to handle TLS certificates in an easy way. The user just needs to setup a boolean to true to do so

### CODE REVIEW

File: stable/drupal/values.yaml [hunk 1]
```diff
@@ -107,27 +107,36 @@ mariadb:
...
-  ##
-  # tls:
-  #   - secretName: wordpress.local-tls
-  #     hosts:
-  #       - drupal.local
+    ## Set this to true in order to enable TLS on the ingress record
+    ## A side effect of this will be that the backend drupal service will be connected at port 443
+    tls: false
+
+    ## Set this to true in order to add the corresponding annontations for cert-manager
```
Comment (tompizmor) [inline stable/drupal/values.yaml]: typo

File: stable/drupal/templates/ingress.yaml [hunk 2]
```diff
@@ -1,28 +1,38 @@
...
 metadata:
-  name: {{ template "drupal.fullname" . }}
+  name: "{{- printf "%s-%s" .name $.Release.Name | trunc 63 | trimSuffix "-" -}}"
   labels:
-    app: {{ template "drupal.fullname" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-    release: "{{ .Release.Name }}"
-    heritage: "{{ .Release.Service }}"
+    app: {{ template "drupal.fullname" $ }}
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
```
Comment (tompizmor) [inline stable/drupal/templates/ingress.yaml]: `{{ template "drupal.chart" $ }}`  ??

**Review by tompizmor — CHANGES_REQUESTED** [review]

File: stable/drupal/templates/ingress.yaml (hunk 2 above)
Comment (juan131) [reply stable/drupal/templates/ingress.yaml]: Done

**Pull request merged**